client/task-list/tasks/payments/RemotePayments/components/PaymentMethod.js

```diff
@@ -55,73 +55,99 @@ export const PaymentMethod = ( {
 		return isOptionsUpdating();
 	} );
 
-	const installStep = useMemo( () => {
-		if ( ! plugins || ! plugins.length ) {
+	const pluginsToInstall = plugins.filter(
+		( m ) => ! activePlugins.includes( m )
+	);
+
+	useEffect( () => {
+		if (
+			pluginsToInstall.length ||
+			paymentGateway ||
+			isFetchingPaymentGateway
+		) {
 			return;
 		}
+		fetchGateway();
+	}, [ pluginsToInstall ] );
 
-		const pluginsToInstall = plugins.filter(
-			( m ) => ! activePlugins.includes( m )
-		);
-		const pluginNamesString = plugins
-			.map( ( pluginSlug ) => pluginNames[ pluginSlug ] )
-			.join( ' ' + __( 'and', 'woocommerce-admin' ) + ' ' );
+	// @todo This should updated to use the data store in https://github.com/woocommerce/woocommerce-admin/pull/6918
+	const fetchGateway = () => {
+		setIsFetchingPaymentGateway( true );
+		apiFetch( {
+			path: 'wc/v3/payment_gateways/' + key,
```

Comment:
Something I was wondering earlier, can we rely on `key` to always be equivalent to the API path?

Reply:
Very good question.  I believe the current keys do not fit.  However, this might be a good opportunity to change that since we won't have any dependencies on those keys.

This may be something we have to come back to later and add an additional `paymentGatewayKey`, but my hope is we won't have to.